gossip/gossip.go

```diff
@@ -777,14 +781,14 @@ func (g *Gossip) bootstrap() {
 			select {
 			case <-bootstrapTimer.C:
 				bootstrapTimer.Read = true
-				// continue
+				// break
```

Comment:
What's going on with these comments? They should probably be removed, but if they were ever uncommented then it looks like the intention is to continue, not break.